libs/data-mapper/src/lib/ui/ReactFlowWrapper.tsx

```diff
@@ -69,6 +68,15 @@ export const ReactFlowWrapper = ({ canvasBlockHeight, canvasBlockWidth }: ReactF
   const [canvasZoom, setCanvasZoom] = useState(defaultCanvasZoom);
   const [displayMiniMap, { toggle: toggleDisplayMiniMap }] = useBoolean(false);
 
+  const nodeTypes = useMemo(
+    () => ({
+      [ReactFlowNodeType.SchemaNode]: SchemaCard,
+      [ReactFlowNodeType.FunctionNode]: useExpandedFunctionCards ? ExpandedFunctionCard : SimpleFunctionCard,
+    }),
+    [useExpandedFunctionCards]
+  );
+  const edgeTypes = { [ReactFlowEdgeType.ConnectionEdge]: ConnectionEdge };
```

Comment:
Assuming the same case as previous comment

Reply:
Changed